taskcluster/ffios_taskgraph/__init__.py

```diff
@@ -0,0 +1,27 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This file is a copy of https://github.com/mozilla-mobile/fenix/blob/34144fd254f00bd8a4176a296fad384ced69f2bf/taskcluster/fenix_taskgraph/__init__.py

I removed the version handling. We just want jobs to run on a dedicated branch. There is no release planned for now.